code/Controllers/ModelAsController.php

```diff
@@ -118,12 +118,11 @@ public function getNestedController(): ContentController
         }
 
         // Select child page
-        $tableName = DataObject::singleton(SiteTree::class)->baseTable();
-        $conditions = [sprintf('"%s"."URLSegment"', $tableName) => $urlSegment];
+        $conditions = ['URLSegment' => $urlSegment];
         if (SiteTree::config()->get('nested_urls')) {
-            $conditions[] = [sprintf('"%s"."ParentID"', $tableName) => 0];
+            $conditions[] = ['ParentID' => 0];
         }
-        $sitetree = DataObject::get_one(SiteTree::class, $conditions);
+        $sitetree = SiteTree::get()->setUseCache(true)->filter($conditions)->first();
```

Comment:
`get_one()` uses raw SQL in a `where` - but there's no need for that, we can just use a safe `filter` instead.
Same with other similar changes in this PR.